src/components/tracked-controls.js

```diff
@@ -145,12 +145,12 @@ module.exports.Component = registerComponent('tracked-controls', {
     // Compose pose from Gamepad.
     pose = controller.pose;
     // If no orientation, use camera.
-    if (pose.orientation) {
+    if (pose && pose.orientation) {
```

Comment:
reading that extension, it says that in some cases position and orientation may/should be null, also pose itself, so the guards seem appropriate?